libparsec/src/workspace.rs

```diff
@@ -404,7 +404,7 @@ pub async fn workspace_mount(
     )))
 }
 
-#[cfg(not(target_arch = "wasm32"))]
+#[cfg(all(not(target_arch = "wasm32"), feature = "mountpoint"))]
```

Comment:
You must also change the `#[cfg]` on the other implementation of this function (see above), right now there is no implementation of this function if not in wasm32 and `mountpoint` feature is disabled

Reply:
reverted